hw/ip/i2c/doc/_index.md

```diff
@@ -142,6 +143,30 @@ Typically every byte transmitted must also receive an ACK signal, and the IP wil
 However, there are some I2C commands which do not require an ACK.
 In those cases this flag should be asserted with FBYTE indicating no ACK is expected and no interrupt should be raised if the ACK is not received. 
 
+### Target Address Registers
+
+I2C target device is assigned two 7-bit address and 7-bit mask pairs.
+The target device accepts a transaction if the result of the bitwise AND operation performed on the transaction address sent by the host and a mask matches the assigned address corresponding to the mask.
+Thus, the target device may respond to any of the two assigned unique addresses.
+The assigned address and mask pairs are set in registers {{< regref ADDRESS0 >}}, {{< regref MASK0 >}}, {{< regref ADDRESS1 >}}, and {{< regref MASK1 >}}. 
+
+### Acquired Formatted Data
+
+This section applies to I2C in the target mode.
+When the target accepts a transaction, it inserts the transaction address, read/write bit, and START signal sent by the host into the ACQ FIFO where they can be accessed by software.
+ACQ FIFO output corresponds to {{< regref ACQDATA >}}.
+If the transaction is a write operation (R/W bit = 0), the target proceeds to read bytes from the bus and insert them into the ACQ FIFO until the host terminates the transaction by sending a STOP or RESTART signal.
+A STOP or RESTART indicator is inserted into the ACQ FIFO as the next entry following the last byte read, in which case other bits are junk.
+If the transaction is a read operation (R/W bit = 1), the target pulls bytes out of TX FIFO and transmits them to the bus until the host signals the end of the transfer by sending a NACK signal.
+TX FIFO input corresponds to {{< regref TXDATA >}}.
```

Comment:
Inserted such a comment.

Reply:
I inserted such a comment.

Reply:
I am wondering "implicitly" holding SCL low when FIFO is empty to stretch the clock would be functional. 
It looks like that any target can stretch clock but what happens if the target is hooked up with the host which does not support or disable that feature via register program?

Reply:
The only other option would be to send junk which seems worse?
As far as I can see the SMBus spec requires it. I2C suggests it is optional but only because some "slave devices do not include an SCL driver" and then goes on to explain how a target would use it.

Reply:
@tunghoang290780: It sounds like you are looking to support a hypothetical use case, that we may or may not need.  @igk could you put a note of warning in the first section that explicitly says this target is only compatible with hosts that support clock stretching?

Reply:
Inserted the note about clock stretching.